### PR TITLE
Decentralize loggerific errorMessages.txt

### PR DESCRIPTION
**Pull request by ngfreiter**
This takes the `errorMessages.txt` file which existed only in `shr-cli` and splits it into multiple files, so that each module has its own `errorMessages.txt` file. Also incorrectly numbered or otherwise broken errors were fixed. Tests that depended on the old error messages format were refactored.

PR 3/7
Affects:
- `shr-cli`
- `shr-text-import`
- `shr-expand`
- `shr-fhir-export`
- `shr-json-schema-export`
- `shr-json-javadoc`
- `shr-es6-export`

### CODE REVIEW

File: lib/expand.js
Comment (cmoesel) [inline lib/expand.js:84]: This is totally fine, but I think that in these cases `basedOn.fqn` and `de.identifier.fqn` would have worked too.  You don't need to change it, I just wanted to be sure you're aware of it.

File: lib/expand.js
Comment (cmoesel) [inline lib/expand.js]:
```javascript
  return require('path').join(__dirname, '..', 'errorMessages.txt');
```

File: test/expand-test.js
Comment (cmoesel) [inline test/expand-test.js:179]: I'm sure this was very annoying to have to actual modify all these tests, but... I really like the way they came out.  Seems much more structured than the previous approach of just looking for substrings.  So that's nice...

**Review by cmoesel — CHANGES_REQUESTED** [review]
Only one minor thing to address (`path.join`).

File: lib/expand.js
Comment (ngfreiter) [reply lib/expand.js]: Fixed this here https://github.com/standardhealth/shr-expand/pull/45/commits/688eb24c41b30429e0bfe179ed027d25110224e0.

**Comment by cmoesel** [comment]
Changes all look good.  Holding off on merging pending a decision on whether we want this in master yet.  See: https://github.com/standardhealth/shr-cli/pull/227#pullrequestreview-280987415

**Review by cmoesel — APPROVED** [review]
Looks good.  I may squash or rebase this to clean up commits before merging.

**Pull request merged**
